report/sections/conclusion.tex

```diff
@@ -1,3 +1,17 @@
+
+\subsection{Further Work}
+A top 100 score on Kaggle is an accomplishment but there is plenty of improvement to be done, we are in that bottom quartile of scores after all. Hyperparameter studies beyond batch sizes and training splits were not performed, it is likely that hyperparameters such as learning rate and learning rate decay could have helped improve the network past current thresholds. A few props were taken from the winning Kaggle submission, what else could be garnered from their solution? A smaller network for one, feeding images of \isize{69} through such a large network might have been the folly of those runs which ultimately led to overfitting, there were simply too many network parameters and not enough input parameters to fill them. Follies with data-augmentation may not have been negligible either, while our images were certainly rotationally invariant, too many translations may have been performed and made the apparent dataset too large for the network to settle on. Tweaking the learning rate and number of epochs could have worked well, though in that scenario --with resources limited as they were-- there might not have been time to sufficiently explore that line of experimentation which would have involved running 8-12 hour jobs. Ultimately, many runs with varying methods producing varying results --instead of trying to laser in on a single great score by perpetually improving a single good result-- may have given a breadth but not depth of material to present. 
+
+With the network as it is, could there be other astronomical problems to tackle? In the coming years the James Webb space telescope will soon be launched (or hopefully, it's already been delayed multiple times this decade). An infrared telescope, it will be able to see objects in the universe which conventional visible light telescopes cannot. This could open up whole new bodies previously obscured by dust and nebulae. Notable here, it will be able to detect the earliest galaxies in our universe. The network could probably classify cats too.
+
+% what else? 
+
+\subsection{Contributions}
+Diego maintained organization and wrote the initial code for training and validation splitting. Ankur gathered the many valuable research materials and implemented early versions of the \vgg architecture. Tristan finalized the implementations and ran the multi-hour experiments on the GPU. Everyone wrote a roughly equal amount of sections in this paper.
+
+\subsection{Acknowledgments}
+
+Gustaf Borgström the teaching assistant assigned to us for keeping us on schedule and advising us on the structure of this paper.
```

Comment:
I think this would also be a good place to acknowledge the author of the reference kernel, the winner of the Kaggle competition, and the authors of the paper "Star-galaxy Classification Using Deep Convolutional Neural Networks".

Reply:
I think we cite the winner sufficiently, it would be the first time mentioning the reference notebook without having any contact with him it may sound strange. 

"James Lawlor, from whom we lifted a lot of our solution from"?